pkg/constants/constants.go

```diff
@@ -1,7 +1,12 @@
 package constants
 
+import "time"
+
 const (
 	ManagedLoggingFinalizer = "logging.managed.openshift.io"
 	HLFWatchedNamespace     = "openshift-logging"
 	OperatorNamespace       = "openshift-hypershift-logging-operator"
+	MintServiceAccountName  = "mint-service-account"
```

Comment:
I think the service account name should be `cloudwatch-audit-exporter` as it is being used in https://gitlab.cee.redhat.com/service/uhc-clusters-service/-/blob/master/pkg/acm/audit_log.go#L20 and https://access.redhat.com/solutions/7002219

Reply:
Fixed